src/main/java/jira/HelloWorld.java

```diff
@@ -0,0 +1,20 @@
+package jira;
+
+import com.atlassian.connect.spring.AtlassianHostUser;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
```

Comment:
what is this class for?

Reply:
I was testing if the project builds and runs with no errors